src/shared/js/ajax.html

```diff
@@ -247,13 +247,18 @@
 			return output;
 		},
 
-		fire: function(type, detail) {
-			if (this._eventPrefix) {
+		fire: function(type, detail, options) {
+			if (this._eventPrefix && DataUtils.getPathValue('prefix', options)) {
```

Comment:
good catch but those refs should be extracted to the IIFE so it reads cleaner. see [commit](https://github.com/MediaMath/strand/commit/d6666791a8160202faecafc601c37f6929971ef5)